src/docs/user/references/VariablesReference.adoc

```diff
@@ -1,140 +1,197 @@
 This section describes useful variables, their scope, and their usage.
 
-[cols="1,1,1,1", options="header"]
+The following script names refer to:
+
+* <<_selection,selection>> : a script executed to select nodes suitable to execute the task. This script is executed on many nodes from the scheduling loop, before the task is deployed.
+* <<_fork_environment, environment>> : the default mode to execute a task is to fork a dedicated JVM from the selected ProActive Node. The environment script, if defined in the workflow, is executed on the node to
```

Comment:
I know that you put spaces before `:` for readability but for me it's aa typo. Also for consistency it would be nice to remove it.